src/site/content/en/learn/css/inheritance/index.md

```diff
@@ -0,0 +1,341 @@
+- [text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)
+- [visibility](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility)
+- [white-space](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space)
+- [widows](https://developer.mozilla.org/en-US/docs/Web/CSS/widows)
+- [word-spacing](https://developer.mozilla.org/en-US/docs/Web/CSS/word-spacing)
+
+## How inheritance works
+
+Every HTML element has every CSS property defined by default with an initial value.
+An initial value is a property that's not inherited and shows up as a default
+if the cascade fails to calculate a value for that element.
+
+<figure class="w-figure">
+{% Video src="video/VbAJIREinuYvovrBzzvEyZOpw5w1/OvoYqOMcdFZL7wJQIL2C.mp4" %}
+</figure>
+
+Properties that can be inherited cascade downwards,
+and child elements will get a computed value which represents its parent's value.
+This means that if a parent has `font-weight` set to `bold` all child elements will be bold,
+unless their `font-weight` is set to a different value,
+or the user agent stylesheet has a value for `font-weight` for that element.
+
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'xxgGPOZ'
+} %}
+</figure>
+
+## How to explicitly inherit and control inheritance
+
+Inheritance can affect elements in unexpected ways so CSS has tools to help with that.
+
+### The `inherit` keyword
+
+You can make any property inherit its parent's computed value with the `inherit` keyword.
+A useful way to use this keyword is to create exceptions.
+
+```css
+strong {
+  font-weight: 900;
+}
+```
+
+This CSS snippet sets all `<strong>` elements to have a `font-weight` of `900`,
+instead of the default `bold` value.
+
+```css
+.my-component {
+  font-weight: 500;
+}
+```
+
+The `.my-component` class sets `font-weight` to `500` instead.
+To make the `<strong>` elements inside `.my-component` also `font-weight: 500` add:
+
+```css
+.my-component strong {
+  font-weight: inherit;
+}
+```
+
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'eYgNedO'
+} %}
+</figure>
+
+Now, the `<strong>` elements inside `.my-component` will have a `font-weight` of `500`.
+
+You could explicitly set this value,
+but if you use `inherit` and the CSS of `.my-component` changes in the future,
+you can guarantee that your `<strong>` will automatically stay up to date with it.
+
+### The `initial` keyword
+
+Inheritance can cause problems with your elements and `initial` provides you with a powerful reset option.
+
+You learned earlier that every property has a default value in CSS.
+The `initial` keyword sets a property back to that initial, default value.
+
+```css
+strong {
+  font-weight: initial;
+}
+```
+
+This snippet will remove the bold weight from all `<strong>` elements and instead,
+make them normal weight, which is the initial value.
+
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'OJWVORZ'
```

Comment:
Maybe on the second paragraph reference "because I am within the aside element" to show the difference between them

Reply:
Yeh I've fleshed that one out now and it's much better